src/main/java/com/project/trav/domain/entity/Ticket.java

```diff
@@ -0,0 +1,31 @@
+package com.project.trav.domain.entity;
+
+import lombok.*;
```

Comment:
Please, use separate imports instead of `*`. You can configure IDE to not replace imports with `*`. Check this in other files, too